src/tools/interop/idt/README.md

```diff
@@ -0,0 +1,564 @@
+# Interoperability Debugging Tool
+
+## Overview
+
+The “Interoperability Debugging Tool” (IDT) is a python-based tool that supports
+a variety of commands that are useful in the context of interoperability testing
+of Matter devices and app controllers.
+
+### Discovery
+
+While in discovery mode, the tool displays all Matter devices that are in
+commission and/or operational mode. This is useful to have a clear understanding
+of all Matter devices currently “active” in the testing environment.
+
+See section “4.3. Discovery” of the Matter spec for official documentation.
+
+When run interactively, discovery functions in one of two modes: BLE and DNS-SD.
+
+### Capture
+
+While in capture mode, the tool starts capturing all data of interest (e.g.
+video recording of interactions with the mobile app, logs from all components
+involved, network packets capture, etc.) while a test is being conducted
+manually. It also provides feedback to the user on test setup and execution.
+
+When the test completes, capture mode is stopped and all captured data is zipped
+in a file that can then be sent to all parties involved in investigating any
+issue uncovered via the manual test. Each ecosystem may implement an analysis
+that analyzes capture data, displays info to the user, probes the local
+environment and generates additional artifacts.
+
+## Getting started
+
+## Raspberry Pi installation
+
+### Environment overview
+
+The execution environment of IDT when using Raspberry Pi is shown in the figure
+below.
+
+[TODO] add figure.
+
+The Raspberry Pi is referred to as the "target environment" in this document. It
+is where all the "discovery" and "capture" work is performed.
+
+The "admin" computer is the machine used to connect to and control the RPi, and
+to fetch artifacts which were created during capture from the RPi.
+
+This directory contains tools for use on both the admin computer and the target
+environment.
+
+### Environment details
+
+1. This guide and the `idt` scripts expect **_one_** instance of `idt` on both
+   the admin computer and the target environment.
+1. The expected install location on the target environment is the home directory
+   of the user specified in `idt/scripts/vars.sh`. Default values are:
+    ```
```

Comment:
I've been using the [Markdown lint VSCode extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) recently. It looks like you have some warnings, such as ```bash here.

Reply:
Thanks, removed!

There was a CI job that complained about `commandline` annotation for a codeblock (now removed) but it didn't complain about `bash`.